am-lib/src/test/java/integration/uk/gov/hmcts/reform/amlib/GrantAccessIntegrationTest.java

```diff
@@ -75,4 +77,29 @@ void whenCreatingDuplicateResourceAccessEntryIsOverwritten() {
 
         assertThat(databaseHelper.countExplicitPermissions(resourceId)).isEqualTo(1);
     }
+
+    @Test
+    void whenGrantingAccessForMultipleUsersEntriesShouldAppearInDatabase() {
+        Map<JsonPointer, Set<Permission>> multipleAttributePermissions = new ConcurrentHashMap<>();
+        multipleAttributePermissions.put(JsonPointer.valueOf("/claimant"), EXPLICIT_READ_CREATE_UPDATE_PERMISSIONS);
+
+        service.grantExplicitResourceAccess(createGrant(resourceId, MULTIPLE_ACCESSOR_IDS,
+            multipleAttributePermissions));
+
+        assertThat(databaseHelper.findExplicitPermissions(resourceId)).hasSize(3)
+            .extracting(ExplicitAccessRecord::getAccessorId).contains("a","b","c");
+    }
+
+    @Test
+    void whenGrantingAccessForMultipleUsersAndMultipleAttributesEntriesShouldAppearInDatabase() {
```

Comment:
What value does the test have on top of the previous one? Does it cover new execution path?

Reply:
Took out before this comment made :D